en-US/firefox-ios.xliff

```diff
@@ -1132,6 +1121,96 @@
         <target>Forward</target>
         <note>Accessibility Label for the tab toolbar Forward button</note>
       </trans-unit>
+      <trans-unit id="FxA.AccountVerificationDetails">
+        <source>Wrong email? Disconnect below to start over.</source>
+        <target>Wrong email? Disconnect below to start over.</target>
+        <note>Label stating how to disconnect account</note>
+      </trans-unit>
+      <trans-unit id="FxA.AccountVerificationRequired">
+        <source>Account Verification Required</source>
+        <target>Account Verification Required</target>
+        <note>Label stating your account is not verified</note>
+      </trans-unit>
+      <trans-unit id="FxA.AccountVerificationRequiredSurface">
+        <source>You need to verify %@. Check your email for the verification link from Firefox.</source>
```

Comment:
The verification link is not really from Firefox (it's from Firefox Account), but I guess there was copy review for this.